common/script/libs/achievements.js

```diff
@@ -0,0 +1,507 @@
+import i18n from '../i18n';
+
+let achievs = {};
+let index;
+
```

Comment:
Ohh @crookedneighbor I see what you're saying!  Yeah, that makes sense.  I can do that.